renovate.json

```diff
@@ -0,0 +1,5 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
```

Comment:
Is it okay to introduce renovate for this project? Will we get flooded with PRs?